dig.go

```diff
@@ -213,21 +213,38 @@ type VisualizeOption interface {
 	unimplemented()
 }
 
-var _graphTmpl = template.Must(template.New("DotGraph").Parse(`digraph {
+var funcMap = template.FuncMap{
+	"quote": quote,
```

Comment:
optional: can inline this instead of declaring a new global variable.

```go
var _graphTmpl = template.Must(
  template.New("DotGraph").
    Funcs(template.FuncMap{
    }).
    Parse(`
`),
)
```

(Note that the lines are ending with `.` to indicate that more for this expression is going to follow on the next line.)

Reply:
Combined this and next comment and made the template a single expression 😄